### ISSUE TITLE
Navigation menu is covering up integration workflow 

### ISSUE THREAD

**Issue by dongniwang** [syndesisio/syndesis-ui#850]
I was in the create an integration workflow and wanted to open up the navigation menu to check on something. 

As the menu opened up, it also covered part of integration workflow interface. My understanding is that opening up the navigation menu would push the integration visualization pane to the right without laying on top of the visualization pane. 

Is this intentional or it's just a bug? 

![image](https://user-images.githubusercontent.com/24943812/29787345-8a976660-8bfc-11e7-95de-f80a755a9763.png)

**Comment by gashcrumb** [comment]
yeah, the trouble here is at the moment the nav menu is a jquery plugin.  There's some hacks in place to get it to hide and show as needed and adjust the main container width, but since it's not fully integrated with angular there's this edge case.

**Comment by akieling** [comment]
@dongniwang shall we fix this one now or leave it to after TP1?

**Comment by sjcox-rh** [comment]
@akieling @gashcrumb Im okay leaving it as is for TP1 because I know there is a lot left to do. Just want to make sure this issue doesn't get lost and is resolved as some point in the future.  @dongniwang, you okay with that?

**Comment by dongniwang** [comment]
Agreed, we can come back and fix this after TP1. Thanks!

**Comment by astefanutti** [comment]
Let me works on it and see what I can do as I refactored the main integration workflow layout so that it should be easier.
